components/team_card/team_card_storage.lua

```diff
@@ -0,0 +1,92 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:TeamCard/Storage
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Custom = require('Module:TeamCard/Custom')
+local String = require('Module:StringUtils')
+-- TODO: Once the Template calls are not needed (when RL has been moved to Module), deprecate Qualifier
+local Qualifier = require('Module:TeamCard/Qualifier')
+local Variables = require('Module:Variables')
+
+local TeamCardStorage = {}
+
+function TeamCardStorage.saveToLpdb(args, teamObject, players, playerPrize)
+	local team, teamTemplate
+	local image, imageDark
+
+	if type(teamObject) == 'table' then
+		if teamObject.team2 or teamObject.team3 then
+			team = 'TBD'
+		else
+			teamTemplate = teamObject.teamtemplate
+			team = teamObject.lpdb
+			image = args.image1
+			imageDark = args.imagedark1
+		end
+	end
+
+	local smwPrefix = args.smw_prefix or args['smw prefix'] or
+                      Variables.varDefault('smw prefix', Variables.varDefault('smw_prefix', ''))
```

Comment:
What's up with `smw prefix` vs `smw_prefix`? Now would be a good time to rename the variable in general

Reply:
Most likely different wikis using different. The module is used by all but 5 wikis (RL, CS, WC, SC, SC2). Can look into standardizing it, but feels more like something for the complete rework.

Reply:
where do those vars even get set?
prize pools?

Reply:
Yes
eg https://liquipedia.net/leagueoflegends/index.php?title=Template:Prize_pool_start&action=edit#mw-ce-l14

Reply:
Looking at botting `smw prefix` to `smw_prefix` so there's only one variant instead of two.

Reply:
Removed `smw prefix` from this PR. Botting is about 65% done.